### PR TITLE
Add initial data load readme

### PR DESCRIPTION
**Pull request by srezacova**
## Description

Add readme for Initial data load service.

### CODE REVIEW

File: initial-data-load/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+| LOG_LABEL                          | no       | oscar-initial-data-upload                                          | Title of the logs that you can see in the terminal when you run the service.                                                                                                                                                                                                                                     |
+| AZURE_STORAGE_CONNECTION_STRING    | yes      | -                                                                  | Default Azurite connection string from documentation used for a local development.                                                                                                                                                                                                                               |
```
Comment (bariela) [inline initial-data-load/README.md:10]: I would not describe it as default. It's simply 'Connection string for Azure Storage (in cloud or local Azurite)

File: initial-data-load/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+| AZURE_STORAGE_LAYER_CONTAINER_NAME | yes      | layer-geo-data                                                     | Name of the Blob Container to which geojsons are saved in the Azurite.                                             |
+| MONGO_URI                          | yes      | mongodb://localhost:27917/oscar or mongodb://localhost:27017/oscar | URI with port on which the MongoDB is running. In the development environment it runs on port 27917 and in the testing environment on port 27017.                                                                                                                                                                |
+| UPLOAD_DATA_TYPES                  | no       | -                                                                  | Data sets to be uploaded into the database. There are two options: 'mapLayers' for the map view configuration and 'attributes' for application data.                                                                                                                                                             |
```
Comment (bariela) [inline initial-data-load/README.md:13]: Is it clear that you can use both in the same run?

File: initial-data-load/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| COUNTRY                            | yes      | ''                                                                 | Name of the folder from which you get the data for the specific country.                                                                                                                                                                                                                                         |
+| NEW_STORAGE_CONTAINERS             | no       | ''                                                                 | Names of the Blob Containers in the Azurite.                                                                                                                                                                                                                                                                     |
```
Comment (bariela) [inline initial-data-load/README.md:15]: Names of the Blob Containers to be created in Azure Storage.

File: initial-data-load/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+| NEW_STORAGE_QUEUES                 | no       | ''                                                                 | Names of the Storage Queues in the Azurite.                                                                                                                                                                                                                                                                      |
```
Comment (bariela) [inline initial-data-load/README.md:16]: Also to be created, please

**Review by bariela — COMMENTED** [review]
Proposing small wording changes

File: initial-data-load/README.md (hunk 1 above)
Comment (srezacova) [reply initial-data-load/README.md:10]: I fixed this.

File: initial-data-load/README.md (hunk 2 above)
Comment (srezacova) [reply initial-data-load/README.md:13]: I fixed this.

File: initial-data-load/README.md (hunk 3 above)
Comment (srezacova) [reply initial-data-load/README.md:15]: I fixed this.

File: initial-data-load/README.md (hunk 4 above)
Comment (srezacova) [reply initial-data-load/README.md:16]: I fixed this.

**Pull request merged**